e2e/self_node_remediation_test.go

```diff
@@ -123,6 +124,25 @@ var _ = Describe("Self Node Remediation E2E", func() {
 					})
 				})
 
+				Context("OutOfService Remediation Strategy", func() {
```

Comment:
I see that you've added the same test both in context of Worker remediation and of Control Plane remediation.
- IMO it's enough to leave it just in the context of worker remediation
- I see that for this use case (worker remediation) we don't check the env variable in order to skip the test when needed, is that intended ?

Reply:
Thank you for the comment.
> IMO it's enough to leave it just in the context of worker remediation
I agree with you.
> I see that for this use case (worker remediation) we don't check the env variable in order to skip the test when needed, is that intended ?
That's not intended. We need to check both of the use cases. I will fix it.